pkg/validation/validate_bundle.go

```diff
@@ -34,11 +35,29 @@ func NewBundleValidator(b *bpb.ClusterBundle) *BundleValidator {
 // Validate validates Bundles, providing as many errors as it can.
 func (b *BundleValidator) Validate() []error {
 	var errs []error
+	errs = append(errs, b.validateBundle()...)
 	errs = append(errs, b.validateComponentPackageNames()...)
 	errs = append(errs, b.validateClusterObjNames()...)
 	return errs
 }
 
+func (b *BundleValidator) validateBundle() []error {
+	var errs []error
+	n := b.Bundle.GetMetadata().GetName()
+	if n == "" {
+		errs = append(errs, fmt.Errorf("bundle name was empty, but must always be present"))
+	}
+	api := b.Bundle.GetApiVersion()
+	if !strings.Contains(api, "gke.io/k8s-cluster-bundle/") {
```

Comment:
The error message (and the comment for the bundle, https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/blob/master/pkg/apis/bundle/v1alpha1/cluster_bundle.proto#L28) say that the format is 

"gke.io/k8s-cluster-bundle/<version>"

So, 'Contains' will match strings like "foogke.io/k8s-cluster-bundle/bar" or "gke.io/k8s-cluster-bundle/" (which doesn't have the version part)

So, perhaps something like this: 
ver := "gke.io/k8s-cluster-bundle/"
if !(strings.Index(api, ver) == 0 && len(api) > len(ver))  {
   ....
}

Or, with a regex (https://play.golang.org/p/v17peCDPOg7): 
matched, err := regexp.MatchString("^gke\\.io\\/k8s\\-cluster-bundle\\/[a-zA-Z0-9]+$", api) 
if !matched || err != nil {
   ....
}

(regex doesn't enforce v1alpha1/v1beta1/v1, etc, just alphanumeric versions)

Reply:
Good point; I got lazy =)

Reply:
Done